Makefile.sub

```diff
@@ -71,6 +71,7 @@ jlm-release: libjlm-release libjlc-release jlm-print-release jlm-opt-release jlc
 jlm-clean:
 	@rm -rf $(JLM_BUILD)
 	@rm -rf $(JLM_BIN)
-	@rm -rf $(JLM_ROOT)/utests.log
-	@rm -rf $(JLM_ROOT)/ctests.log
-	@rm -rf $(JLM_ROOT)/check.log
+	@rm -f $(JLM_ROOT)/utests.log
+	@rm -f $(JLM_ROOT)/ctests.log
+	@rm -f $(JLM_ROOT)/check.log
+	@rm -f $(LLVMPATHSFILE)
```

Comment:
LLVMPATHSFILE should be clean up in libjlc/Makefile.sub, no?

Reply:
Since the compilation now happens in build, I didn't view it as necessary to call the clean for each individual component. But there are a few other files that get generates, like the test logs and the llvmpaths.hpp.
So jlm-clean, cleans everything by deleting build, bin, and all other temp files.

We could change this, so it calls all the individual parts and then simply does a rmdir build bin
Another option could be to remove the individual clean commands